content/en/real_user_monitoring/guide/send-rum-custom-actions.md

```diff
@@ -16,8 +16,10 @@ Real User Monitoring [automatically collects actions][1] on your web application
 To create a new RUM action, use the `addAction` API. Give your action a name and then attach context attributes in the form of a JavaScript object. In the following example, a `checkout` action is created with details about the user cart when the user clicks on the checkout button.
 
 ```javascript
+import { datadogRum } from '@datadog/browser-rum';
+
 function onCheckoutButtonClick(cart) {
-    DD_RUM.addAction('checkout', {
+    datadogRum.addAction('checkout', {
```

Comment:
As detailed in the [action doc](https://docs.datadoghq.com/real_user_monitoring/browser/tracking_user_actions/?tab=npm#custom-actions), the syntax depends on your setup (NPM / CDN async / CDN sync).
Adding the different versions in this guide could be a nice improvement.